extras/indexing/src/test/java/org/apache/rya/indexing/external/PrecompJoinOptimizerIntegrationTest.java

```diff
@@ -387,7 +387,7 @@ public void testEvaluateTwoIndexUnionFilter() throws AccumuloException,
 		pcjConn.prepareTupleQuery(QueryLanguage.SPARQL, queryString).evaluate(
 				crh);
 
-		Assert.assertEquals(6, crh.getCount());
```

Comment:
This supposed to be 6.  I guess it will change when we merge your IT name change PR in.